docs/contributing/README.md

```diff
@@ -95,16 +97,18 @@ For additional features and advanced configurations, refer to the official [MkDo
 
 ## Testing
 
+=== "NVIDIA CUDA"
+
```

Comment:
I don't think this does anything unless this entire section is indented, which I'd rather not do. Would you be happy to remove this, or replace it with a sentence saying that the commands below are for CUDA?

Reply:
of course, added a comment instead